code/clue.js

```diff
@@ -1,24 +1,61 @@
 // CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// const mrGreen = {
+//   firstName: "Jacob",
+//   lastName: "Green",
+//   color: "green",
+//   description: "He has a lot of connections",
+//   age: 45,
+//   image: "assets/green.png",
+//   occupation: "Entrepreneur"
+// }
 
-const mrGreen = {
-  firstName: "Jacob",
-  lastName: "Green",
-  color: "green",
-  description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
-  occupation: "Entrepreneur"
+
+
+ function Suspect(firstName, lastName, color, description, age, occupation, image) {
+  this.firstName = firstName;
+  this.lastName  = lastName;
+  this.color = color;
+  this.description = description;
+  this.age = age;
+  this.occupation = occupation;
+  this.image = image;
 }
 
+var mrGreen = new Suspect("Jacob", "Green", "green", "He has a lot of connections", 45, "Entrepreneur", "assets/green.png" );
+var profPlum = new Suspect("Victor", "Plum", "purple", "He is a quick-witted", 40, "Professor", "assets/plum.png" );
+var missScarlet = new Suspect("Cassandra", "Scarlet", "red", "She is young and attractive", 30, "Chef", "assets/scarlet.png"  );
+var mrsPeacock = new Suspect("Eleanor", "Peacock", "blue", "She nearly always maintains her dignity ", 55, "Retired", "assets/peacock.png"  );
+var mrMustard = new Suspect("Colonel ", "Mustard", "yellow", "He is a dangerous military man", 60, "Military man", "assets/mustard.png"  );
+var mrsWhite = new Suspect("Mrs", "White", "lightgrey", "He does not have a lot of connections", 35, "Servant", "assets/white.png"  );
+
+
+
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS THE THE WEAPONS IF YOU LIKE.
+// //const rope = {
+//   name: "rope",
+//  weight: 10
+// }
+
+function Weapon(name, weight, image,) {
+  this.name = name;
+  this.weight = weight;
+  this.image = image;
+} 
+
+var rope = new Weapon("Rope",0.5, "assets/rope.png");
+var knife = new Weapon("Knife",0.3, "assets/knife.png");
+var candlestick = new Weapon("Candlestick",0.9, "assets/candlestick.png");
+var dumbell = new Weapon("Dumbell",5, "assets/dumbell.png");
+var poison = new Weapon("Poison",0.01, "assets/poison.png");
+var axe = new Weapon("Axe",3, "assets/axe.png");
+var bat = new Weapon("Bat",1.2, "assets/bat.png");
+var trophy = new Weapon("Trophy",3, "assets/trophy.png");
+var pistol = new Weapon("Pistol",0.8, "assets/pistol.png");
```

Comment:
Interesting that you have wrote it like this (in comparence to the out commented code in the beginning om the code). Nothing wrong with that! I'm just curious the reason behind it?